wiki/Gameplay/Accuracy/de.md

```diff
@@ -13,7 +8,7 @@ Die Genauigkeit ist ein prozentualer Messwert für die Fertigkeit eines Spielers
 
 ### ![](/wiki/shared/mode/osu.png) osu!
 
-![Genauigkeit = (50 \* Anzahl der 50er + 100 \* Anzahl der 100er + 300 \* Anzahl der 300er) / 300(Anzahl der 0er + Anzahl der 50er + Anzahl der 100er + Anzahl der 300er)](img/accuracy_osu.png "Genauigkeitsformel für osu!")
+![Genauigkeit = (300 \* Anzahl der 300er + 100 \* Anzahl der 100er + 50 \* Anzahl der 50er) / 300(Anzahl der 300er + Anzahl der 100er + Anzahl der 50er + Anzahl der Misses)](img/accuracy_osu.png "Genauigkeitsformel für osu!")
```

Comment:
```suggestion
![Genauigkeit = (300 \* Anzahl der 300er + 100 \* Anzahl der 100er + 50 \* Anzahl der 50er) / (300 \* (Anzahl der 300er + Anzahl der 100er + Anzahl der 50er + Anzahl der Misses))](img/accuracy_osu_updated.png "Genauigkeitsformel für osu!")
```
Mathematically speaking, this formula should still be correct without the brackets, but when looking at problems like [this one](https://people.math.harvard.edu/~knill/pedagogy/ambiguity/index.html), I think it's better to put everything in brackets, to minimise ambiguity.